appengine/go11x/cloudsql/go.mod

```diff
@@ -1,5 +1,5 @@
 module cloudsql
 
-go 1.11
+go 1.15
```

Comment:
Please keep the `go.mod` versions the same. They specify the minimum version of Go for the module.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#go-version-in-gomod-files (new from a couple weeks ago).

Reply:
Sorry, I remember you told me this last time, but I forgot. This is now done.